neuralpp/test/quick_tests/symoblic/solver_compare_test.py

```diff
@@ -0,0 +1,61 @@
+"""
+Test cases comparing Z3 and SymPy solver.
+"""
+import pytest
+
+from sympy import symbols, Poly, PolynomialError, Interval, oo
+from sympy.solvers.inequalities import solve_rational_inequalities
+from z3 import *
```

Comment:
In general it's best to avoid `from package import *` as it pollutes the current namespace and make it hard for reader to keep track of the origin of the methods

Reply:
Yeah that makes sense. Fixed.